### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by declanleeler**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -192,6 +194,16 @@ Format: `listo`
...
 * `listo`
 
+### Listing all orders: `incompleteo`
+
+Shows a list of all incomplete orders in ReadyBakey before and during a given date and time
+
+Format: `incompleteo DELIVERYDATETIME`
+
+Examples:
+* `incompleteo 25-12-2022 15:30`
```
Comment (punnyhuimin) [inline docs/UserGuide.md:204]: Explain what this returns?

**Review by punnyhuimin — COMMENTED** [review]
Just a small update to be made.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -79,24 +79,28 @@ Format: `help`
 
 Adds a customer to ReadyBakey’s contact list
 
-Format: `addp n/NAME p/PHONE_NUM a/ADDRESS`
+Format: `addp n/NAME p/PHONE e/EMAIL a/ADDRESS [t/TAG]`
 
 Examples:
 * `addp n/John Doe p/87654321 a/21 Kent Ridge Drive`
```
Comment (darshan152) [inline docs/UserGuide.md:85]: Will this example work? Is email non compulsory? if so we should put it in [] like tags right?

Comment (declanleeler) [reply docs/UserGuide.md:85]: Good catch, removed it

File: docs/UserGuide.md (hunk 1 above)
Comment (declanleeler) [reply docs/UserGuide.md:204]: Got it, I'll explain how the format works

**Review by punnyhuimin — APPROVED** [review]
LGTM

**Pull request merged**